From 060ffcc95b7106c6d174017d9c2dff08c6126aac Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 30 May 2014 14:56:26 +0000
Subject: [PATCH 0471/1000] MANTIS 0020990 extension;
 Platform_Extension_CUSS_1_2: Periodic/Automatic restart of the application

appcontrol.idl
V1.3

- added an attribute for the memory usage in the data structure.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@682 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/appcontrol.idl | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl b/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
index ed67ab8c..7bfb2c8a 100644
--- a/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
@@ -22,7 +22,7 @@
 
 module appControl
 {
-    const string interface_version = "1.2";
+    const string interface_version = "1.3";
    
     // list of strings
     typedef sequence<string> strings;
@@ -95,8 +95,9 @@ module appControl
     struct data
     {
       string applicationName;
-      unsigned long applicationMemoryUsage;
+      unsigned long applicationIoUsage;
       unsigned long applicationTimeUsage;
+      unsigned long applicationMemoryUsage;
     };
 };
 
-- 
2.41.0.windows.1

